### PR TITLE
Support Graphite retries

### PR DESCRIPTION
**Pull request by knoepfel**
This PR addresses https://github.com/HEPCloud/decisionengine_modules/issues/369.

### CODE REVIEW

File: src/decisionengine_modules/graphite_client.py [hunk 1]
```diff
@@ -25,8 +25,9 @@ def __init__(self, host="fifemondata.fnal.gov", pickle_port=2004, logger=None):
     def send_dict(self, namespace, data, debug_print=True, send_data=True):
         """send data contained in dictionary as {k: v} to graphite dataset
         $namespace.k with current timestamp"""
-        if data is None and self.logger is not None:
-            self.logger.warning("Warning: send_dict called with no data")
+        if data is None:
+            if self.logger is not None:
```
Comment (jcpunk) [inline src/decisionengine_modules/graphite_client.py:29]: I believe the complexity scanning tools prefer a single `if` with an `and` joining the two.

Comment (knoepfel) [reply src/decisionengine_modules/graphite_client.py:29]: But the logic is subtly different--`if data is None` evaluates to `True`, the method must return regardless of whether `self.logger is not None`.  The diff obscures this nuance.

Comment (jcpunk) [reply src/decisionengine_modules/graphite_client.py:29]: Ahh, I see now.

**Review by mambelli — APPROVED** [review]

**Comment by mambelli** [comment]
@knoepfel I think in all cases your graphite-retries should be kept, but I'll let you update. If you edit the file, in AWSOccupancy there is a misspelling shortly after the conflict: "AWS capacity data and metods" -> methods

**Review by mambelli — APPROVED** [review]

**Pull request merged**